templates/quickstart-jira-dc.template.yaml

```diff
@@ -1366,3 +1383,7 @@ Outputs:
     Export: {
       Name: !Join ['', [!Ref 'AWS::StackName', '-EFSCname']]
     }
+  CloudWatchDashboardURL:
```

Comment:
Is there a way to test this? I can't seem to find documentation on Cloudwatch Dashboard `!GetAtt` 🤔

Reply:
Ah. That's because `!GetAtt` queries a cloudformation resource, similar to how we retrieve the DB endpoint address